server/engine/plotting.py

```diff
@@ -217,8 +217,8 @@ def generateReportForTable(table, filename, output_fpath, data_type, bilateral=T
     refs = list()
 
     # Suffixes to detect if a pelvis coordinate is translational or rotational.
-    translate_suffixes = ['_tx', '_ty', '_tz', '_force']
-    rotate_suffixes = ['_tilt', '_list', '_rotation', '_moment']
+    translate_suffixes = ['_tx', '_ty', '_tz', '_force', '_vx', '_vy', '_vz', '_px', '_py', '_pz']
+    rotate_suffixes = ['_tilt', '_list', '_rotation', '_moment', '_mx', '_my', '_mz', '_x', '_y', '_z']

```

Comment:
Are these additional suffixes necessary? Looks like they were only used for columns containing `pelvis`. I could easily be missing something here, but seems like L273-291 and L304 may be covering these cases.

Reply:
Yes, I meant to remove these after coming up with a different solution. Updated.